tests/features/sync_flow.feature

```diff
@@ -0,0 +1,164 @@
+      | TT_BLACKBUTTON | device_f  | BlackButton | service2sy   | testpizza       | NaN               | 4passw0rd             | no    | service2sy | testpizza   | admin_bb      | 4passw0rd   | TP/sync/request |
+
+
+  @ft-syncflow @sf-button-flows @sf-03
+  Scenario Outline: SC3 client push the button and device is registered, but conditions fail
+    Given a Client of "<SERVICE>" and a ThirdParty called "<SERVICEPATH>"
+    When a service and subservice are created in the "ORC"
+      | key                        | value           |
+      | DOMAIN_NAME                | admin_domain    |
+      | DOMAIN_ADMIN_USER          | cloud_admin     |
+      | DOMAIN_ADMIN_PASSWORD      | password        |
+      | NEW_SERVICE_NAME           | <SERVICE>       |
+      | NEW_SERVICE_ADMIN_USER     | <SERVICE_ADMIN> |
+      | NEW_SERVICE_ADMIN_PASSWORD | <SERVICE_PWD>   |
+      | KEYPASS_PORT               | 8080            |
+      | KEYSTONE_PORT              | 5001            |
+      | NEW_SUBSERVICE_NAME        | <SERVICEPATH>   |
+    And the "ORC" receive the request "SERVICE_ENTITY" and action "CREATE"
+      | SERVICE_NAME          | <SERVICE>         |
+      | SERVICE_USER_NAME     | admin_bb          |
+      | SERVICE_USER_PASSWORD | 4passw0rd         |
+      | SUBSERVICE_NAME       | <SERVICEPATH>     |
+      | ENTITY_TYPE           | service           |
+      | ENTITY_ID             | <SERVICE>         |
+      | SERVICE_NAME          | <SERVICE>         |
+      | ATT_NAME              | <SERVICEPATH>     |
+      | ATT_PROVIDER          | QA Testpizza CORP |
+      | PROTOCOL              | TT_BLACKBUTTON    |
+      | ATT_ENDPOINT          | <TP_URL>          |
+      | ATT_METHOD            | POST              |
+      | ATT_AUTHENTICATION    | context-adapter   |
+      | ATT_MAPPING           | []                |
+      | ATT_INTERACTION_TYPE  | synchronous       |
+      | ATT_TIMEOUT           | 120               |
+    And a valid token is retrieved for user "<SERVICE_ADMIN>" and password "<SERVICE_PWD>"
+    And a device should be provisioned for service and subservice with certain fields
+      | attribute             | value           |
+      | PROTOCOL              | TT_BLACKBUTTON  |
+      | DEVICE_ID             | <DEVICE_ID>     |
+      | ENTITY_TYPE           | <ENTITY_TYPE>   |
+      | SERVICE_NAME          | <SERVICE>       |
+      | SUBSERVICE_NAME       | <SERVICEPATH>   |
+      | SERVICE_USER_NAME     | <SERVICE_ADMIN> |
+      | SERVICE_USER_PASSWORD | <SERVICE_PWD>   |
+      | TOKEN                 | <TOKEN>         |
+    Then device "<DEVICE_ID>" should be listed under service and subservice
+    #And the button "<DEVICE_ID>" pressed in mode "synchronous" the IOTA should not receive the request "<BT_REQUEST>"
```

Comment:
To be used in further commits. For now, response to failured processing of messages exits with a timeout after pending for a few seconds, and real results can be only seen in log files.